ProductMatrix/files/default.css

```diff
@@ -1,10 +1,12 @@
 div.productmatrix {
+	width:200px;
```

Comment:
This is on purpose to have the version and the status close enough for readability. It looks about the same as the previous version. Without this, the table is not easy to read.